modal/functions.py

```diff
@@ -132,7 +152,16 @@ async def create(
         function_call_id = response.function_call_id
 
         if response.pipelined_inputs:
-            return _Invocation(client.stub, function_call_id, client)
+            input = response.pipelined_inputs[0]
```

Comment:
Can there ever be more than one pipelined input, I assume no, but could we add an assertion that the response will not have more than one in this case?

Reply:
Yea, good point. Will add!

Reply:
Added.